Gemfile.lock

```diff
@@ -161,7 +161,7 @@ GEM
       mime-types (>= 1.16, < 4.0)
       netrc (~> 0.8)
     rexml (3.2.5)
-    rollbar (3.3.2)
+    rollbar (3.3.0)
```

Comment:
Why the regresssion?

Reply:
This was an automatic change. Will add a note to review in next change.